test/tests/treeview-1a.js

```diff
@@ -0,0 +1,675 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertAriaLabelledby = require('../util/assertAriaLabelledby');
+const assertRovingTabindex = require('../util/assertRovingTabindex');
+
+const exampleFile = 'treeview/treeview-1/treeview-1a.html';
+
+const ex = {
+  treeSelector: '#ex1 [role="tree"]',
+  treeitemSelector: '#ex1 [role="treeitem"]',
+  groupSelector: '#ex1 [role="group"]',
+  folderSelector: '#ex1 [role="treeitem"][aria-expanded]',
+  topLevelFolderSelector: '#ex1 [role="tree"] > [role="treeitem"]',
+  nextLevelFolderSelector: '[role="group"] > [role="treeitem"][aria-expanded]',
+  docSelector: '#ex1 .doc[role="treeitem"]',
+  textboxSelector: '#ex1 #last_action'
+};
+
+const openAllFolders = async function (t) {
+  const closedFoldersSelector = ex.treeitemSelector + '[aria-expanded="false"]';
+  let folders = await t.context.session.findElements(By.css(closedFoldersSelector));
+
+  // While their are closed folders
+  while (folders.length != 0) {
```

Comment:
`>`, just so we don't get any weird falsey comparisons in the future. For example, `undefined != 0` is `true`, which will yield a lot of looping :)